app/assets/javascripts/sumofus/backbone/form_methods.js

```diff
@@ -64,6 +65,32 @@ const FormMethods = {
     this.$(`.${plugin_type}-bar__welcome-name`).text(member.welcome_name);
     this.$(`.${plugin_type}-bar__welcome-text`).removeClass('hidden-irrelevant');
   },
+
+  insertActionKitId(form_type, urlParams) {
+    let $form;
+    if(form_type == 'petition') {
+      $form = $('.petition-bar__main').find('form')[0];
```

Comment:
a couple things here -
- in backbone, use `this.$` instead of straight `$` to scope your DOM query to subelements of the element that the backbone view works with.
- in jquery, if you call `[0]`, it nonintuitively returns the first dom element, but not wrapped in a jquery object (so you can't call jquery methods on it). I think you're looking for `.first()`, though it's common to just carry around all matches.
- I don't think it's necessary to differentiate between petition and fundraiser here. The form you want always has the class `action` per `_form.liquid`, so I think this whole block could just become `let $form = this.$('form.action');`